demo/src/Components/Cards/Icon.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Icon = () => (
+  <div>
+    <span>
+      <i className="c-icon" aria-hidden="true" />
```

Comment:
Should be an SVG rather than base-64.  I've seen other frameworks use `<i>` tags for Icons and I don't think it's a great idea.
It's good to use the base64 stuff with icons that never change e.g. a select-box will always have a chevron likewise with a search input.  However if it's likely the icon will be changed we should just expect the user to put in an SVG themselves along with the text.
Having an Icon component that wraps the SVG though is absolutely a good idea.